docs/paper/paper.md

```diff
@@ -21,41 +21,40 @@ link-bibliography: false
 
 Here, we have developed `PlasticParcels` to unify plastic dispersion modelling into one easy-to-use code. While `PlasticParcels` has been designed for researchers who routinely perform plastic particle dispersion simulations, it is equally useful to novice users who are new to Lagrangian ocean analysis techniques.
 
 # Description of the software
-`PlasticParcels` has been designed as a layer over the `Parcels` Lagrangian framework [@Lange2017,@Delandmeter2019]. The core functionality of `Parcels` are its `FieldSets`, `Kernels` and `ParticleSets`. These objects are designed to be as flexible and customisable as possible so that users can perform Lagrangian simulations of a wide variety of particulates, such as tuna, plastic, plankton, icebergs, turtles **(etc. etc. + CITATIONS)**. However, due to the flexible nature of the software, there is a steep learning curve for new users, who often find it difficult to setup their simulations in a rapid fashion due to the complexity of modern hydrodynamic model output. We have developed `PlasticParcels` as user-friendly tool specifically designed for easy-to-generate plastic dispersal simulations. While `PlasticParcels` is primarily designed for use in the cloud and in HPC environments (due to the ever increasing size of hydrodynamic datasets generated from ocean general circulation models), it can be easily installed and run on local machines.
+`PlasticParcels` has been designed as a layer over the `Parcels` Lagrangian framework [@Lange2017,@Delandmeter2019]. The core functionality of `Parcels` are its `FieldSet`, `Kernel` and `ParticleSet` objects. These objects are designed to be as flexible and customisable as possible so that users can perform Lagrangian simulations of a wide variety of particulates, such as tuna, plastic, plankton, icebergs, turtles [@Lange2017]. However, due to the flexible nature of the software, there is a steep learning curve for new users, who often find it difficult to setup their simulations in a rapid fashion due to the complexity of modern hydrodynamic model output. We have developed `PlasticParcels` as user-friendly tool specifically designed for easy-to-generate plastic dispersal simulations. While `PlasticParcels` is primarily designed for use in the cloud and in HPC environments (due to the ever increasing size of hydrodynamic datasets generated from ocean general circulation models), it can be easily installed and run on local machines. A schematic of `PlasticParcels` is shown in Fig. \ref{fig:schematic}.
```

Comment:
Also here lowercase `plasticparcels` and `parcels`?

Reply:
Updated to plasticparcels and parcels